contrib/systemd/tmpfiles.d/labgrid.conf

```diff
@@ -1,6 +1,3 @@
 # Labgrid saves files uploaded to the exporter in /var/cache
-# This configuration file creates the directory and is meant as a starting point
-# it is advised to at least change the group from the default for your
-# environment
```

Comment:
Please don't remove this part. This can also be incorrect, since the users of the exporter now need to be part of the labgrid group, where an existing environment may reuse existing group memberships.

Reply:
Ok, re-added.